announcements/2020-Week-25/azure.md

```diff
@@ -0,0 +1,27 @@
+# :zap: This Week in Releases (June 19, 2020):zap:
+
+## Azure v11.3.3
+
+**If you are upgrading from 11.3.1 or 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release adds support for having the control plane and the tenant clusters in a different subnet.
```

Comment:
```suggestion
This release adds support for having the control plane and the tenant clusters in a different subnet, making it possible to... 
```

It might just be my lack of knowledge: But how does customer benefit from having CP and TCs in a different subnet?

Reply:
1. it is a fix for one specific customer that for some reason they need to use a very custom CIDR that we haven't used before and it broke these 3 apps. I can ask @whites11 for more details not sure how to finish "making possible to.." sentence 😕 
2. sounds good, but what happens when there's more than one change? and isn't the link too long? do we need part after # there?

Reply:
"making it possible to use custom CIDR on .." CP or TC? @whites11 is that correct?

Reply:
you're right should just be https://github.com/giantswarm/releases/tree/master/azure/v11.3.3

Reply:
This release adds support to use a different CIDR for the CP and the TCs making it more flexible when the customer needs to connect the TC network with they're own virtual or on-premise networks.